addon/tests/helpers/ajax.js

```diff
@@ -16,7 +16,6 @@ export default function ajax(url) {
       });
     });
   } else {
-    let fetch = require('fetch').default;
-    return fetch(url).then((response) => response.text());
+    return waitForPromise(fetch(url).then((response) => response.text()));
```

Comment:
Are we sure this doesn't break fastboot?

I'm all in favor of switching to global fetch everywhere, but doing that correctly requires some work to make sure the global fetch does all the things that `require('fetch')` has been doing.

Reply:
We can do the monorepo conversion first, and add a fastboot-using test app to be sure.